google.com.verification.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId":"google.com",
+  "providerName":"Google",
+  "serviceId":"verification",
+  "serviceName":"Google GSuite Setup",
+  "version": 1,
+  "logoUrl":"",
+  "description":"Enables a domain to work with Google GSuite",
```

Comment:
should be:
"Enables a domain to work with Google",

Reply:
Done